900.version-fixes/python.yaml

```diff
@@ -101,6 +101,7 @@
 - { name: "python:shiboken",           ver: "2.0.0",                 ruleset: kaos,        incorrect: true }
 - { name: "python:simpleparse",        ver: "3.0.0",                 ruleset: sisyphus,    incorrect: true }
 - { name: "python:simpleparse",                                      ruleset: sisyphus,    untrusted: true } # accused of fake 3.0.0
+- { name: "python:sip",                verpat: "5\\..*",                                   incorrect: true }
```

Comment:
I don't see anything incorrect about sip 5.x

Reply:
While they share the same project name, they don't seem to be the same. Qutebrowser needs both as build dependencies for example

Reply:
They are definitely different versions of the same project.
https://www.riverbankcomputing.com/hg/sip
https://pypi.org/project/sip/#history
Please remove this rule and this PR can be merged.

Reply:
0f4f595 done
I might want to mark versions 4.x.y as legacy in a later step then, but we can do that in another round :+1: